css/localgov-alert-banner.css

```diff
@@ -60,6 +60,9 @@
 .localgov-alert-banner--notable-person {
   background-color: #000;
 }
+.localgov-alert-banner--notable-person .localgov-alert-banner__title {                                                                                        │
+  color: #fff;
+}
```

Comment:
I don't think we should have the `|` there.
Lets change this to 
````css
.localgov-alert-banner .localgov-alert-banner__title {
  color: inherit;
}
````
This is so it applies a fix to all banners, and that they use the colour assigned to it via `.localgov-alert-banner`. and can then be overridden in themes by just appling a color to that class.